### PR TITLE
feat: added template structure

### PR DESCRIPTION
**Pull request by cl-shaharshavit-rnd**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✔                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

**Review by asafMasa — CHANGES_REQUESTED** [review]
return the HELM

File: package.json [hunk 1]
```diff
@@ -39,42 +39,25 @@
...
-    "tsyringe": "^4.6.0"
+    "tsyringe": "^4.6.0",
+    "yargs": "^17.2.1"
   },
   "devDependencies": {
     "@commitlint/cli": "^13.2.1",
     "@commitlint/config-conventional": "^13.2.0",
     "@map-colonies/eslint-config": "^3.0.1",
     "@map-colonies/prettier-config": "0.0.1",
-    "@map-colonies/standard-version-update-helm-version": "^2.0.1",
```
Comment (asafMasa) [inline package.json:67]: revert with Helm

File: src/index.ts [hunk 2]
```diff
@@ -1,28 +1,20 @@
...
-  port: string;
-}
-
-const serverConfig = get<IServerConfig>('server');
-const port: number = parseInt(serverConfig.port) || DEFAULT_SERVER_PORT;
+async function main(): Promise<void> {
+  const app = getApp();
 
-const app = getApp();
+  app.run(process.argv);
```
Comment (asafMasa) [inline src/index.ts]: Need to be async

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/templates/_helpers.tpl [hunk 3]
```diff
@@ -22,41 +22,34 @@ If release name contains chart name it will be used as a full name.
...
+{{- define "ts-script-boilerplate.labels" -}}
+helm.sh/chart: {{ include "ts-script-boilerplate.chart" . }}
+{{ include "ts-script-boilerplate.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
 
-{{/*
```
Comment (asafMasa) [inline helm/templates/_helpers.tpl]: check

File: helm/values.yaml [hunk 4]
```diff
@@ -6,26 +6,25 @@ global:
...
   dockerRegistryUrl: my-registry-url.io
   flavor: openshift
 
 image:
-  repository: ts-server-boilerplate
+  repository: ts-script-boilerplate
   tag: 'latest'
 
+cron:
+  schedule:  '* 0 * * *'
```
Comment (asafMasa) [inline helm/values.yaml]: 0 0 * * *

**Review by asafMasa — APPROVED** [review]

**Review by asafMasa — APPROVED** [review]

**Review by Yossizz — APPROVED** [review]

**Pull request merged**